app/config/featureSwitches/FeatureSwitch.scala

```diff
@@ -22,13 +22,17 @@ sealed trait FeatureSwitch {
 
 object FeatureSwitch {
   val prefix: String = "feature.switch"
-  val listOfAllFeatureSwitches: List[FeatureSwitch] = List(ShowURBanner)
+  val listOfAllFeatureSwitches: List[FeatureSwitch] = List(ShowURBanner, TaxReturnStatusEnumIsOptional)
 }
 
 case object ShowURBanner extends FeatureSwitch {
   override val name: String = s"${FeatureSwitch.prefix}.show-ur-banner"
 }
 
+case object TaxReturnStatusEnumIsOptional extends FeatureSwitch {
+  override val name = s"${FeatureSwitch.prefix}.tax-return-status-enum-is-optional"
+}
+
```

Comment:
Can we name it so it matches the backend feature switch?
```suggestion
case object EnableR5BProcessing extends FeatureSwitch {
  override val name: String = s"${FeatureSwitch.prefix}.enable-r5b-processing"
}
```